php/libraries/SinglePointLogin.class.inc

```diff
@@ -328,6 +331,26 @@ class SinglePointLogin
                     return false;
                 }
 
+		// check if the account is longer active
+		$date = new DateTime();
+		$currentDay = $date->getTimestamp();
```

Comment:
Technically this should be named something like `$currentTime` as it will incorporate the hour, minute, second, etc.